pkgs/tools/inputmethods/fcitx5/fcitx5-rime.nix

```diff
@@ -42,6 +42,7 @@ stdenv.mkDerivation rec {
   rimeDataDrv = symlinkJoin {
     name = "fcitx5-rime-data";
     paths = rimeDataPkgs;
+    postBuild = "mkdir -p $out/share/rime-data";
```

Comment:
how about use preBuild instead post?

Reply:
`symlinkJoin` doesn't invoke preBuild, so we have to use `postBuild`.